src/parser/index.ts

```diff
@@ -117,20 +116,13 @@ export function parseForESLint(code: string, options?: any): ParseResult {
   const parserOptions = normalizeParserOptions(options);
 
   if (
-    isEnableRunes(svelteConfig, parserOptions) &&
     parserOptions.filePath &&
-    !parserOptions.filePath.endsWith(".svelte") &&
-    // If no `filePath` is set in ESLint, "<input>" will be specified.
-    parserOptions.filePath !== "<input>"
+    (parserOptions.filePath.endsWith(".svelte.js") ||
+      parserOptions.filePath.endsWith(".svelte.ts"))
```

Comment:
I think only `js` and `ts` file can be svelte script file. So I think we can make the terms simpler.

https://svelte.dev/docs/svelte/svelte-js-files